5-WebApp-AuthZ/5-1-Roles/README.md

```diff
@@ -273,15 +273,19 @@ public static IServiceCollection AddMicrosoftIdentityPlatformAuthentication(this
         // [removed for] brevity
 }
 
+/*
 // In code..(Controllers & elsewhere)
-[Authorize(Roles = DirectoryViewers")] // In controllers
+    [Authorize(Roles = DirectoryViewers")] // In controllers
```

Comment:
Should it still be Roles? or Policy?

Reply:
According to Barry Dorrans, the `[Authorize(Roles()]` exists just for backwards compatibility, so I think we would be safe to remove that line of comment and stat pushing the developers to use `[Authorize(Policy()]` instead. 

Fixed